README.md

```diff
@@ -1,2 +1,4 @@
-# Ingenious Thoughts
-### Developed using React, Firebase, Axios and Sass
+# Thinker 
+## previously known as:// Ingenious Thoughts
+### Simple application where users can share their thoughts to the World. 
+## Developed using React, Firebase, Sass and Axios with VSCode on Apple M1 Pro.
```

Comment:
Maybe add the link to your live site in your README.md

Reply:
Thanks Char!!! So helpful :)

Reply:
Thank you, I've add the link.